ccvm/mock_test.go

```diff
@@ -51,6 +51,7 @@ vm:
 var mockVMSpec = VMSpec{
 	MemGiB:       3,
 	CPUs:         2,
+	DiskGiB:      60,
```

Comment:
Should this change be in this commit?

Reply:
Yes, otherwise one of the existing unit tests fails.